metagraph/docs/source/api.rst

```diff
@@ -80,7 +80,7 @@ If the ``align`` flag is set, all the alignment options (explained in the alignm
 Searching count and coordinate indexes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Metagraph supports k-mer count-aware and coordinate-aware indexes. If you are querying such an index,
-you can also set ``query_counts=True`` or ``query_coords=True``, respectively. If you try to query an
+you can also set ``weighted_with_counts=True`` or ``query_coords=True``, respectively. If you try to query an
```

Comment:
Maybe it's better to say "abundances" here instead of "weights", since that's what these are measuring. I guess "weights" only makes sense in the context of some objective function.

Reply:
The thing is that this mode actually does not return the counts/abundances. It returns the sum of all abundances for all k-mers from the query matched to the label. So, it's a weighted sum, similar to the simple query with --count-labels, where each match is weighted with the k-mer's abundance.

Reply:
The other mode (added in this PR) for querying individual abundances/counts is triggered by `--query-counts`, which was previously triggering this weighted sum thing.

Reply:
Maybe we should rename this to `--abundance-sum` or something like that? What's your suggestion?

Reply:
I like that, it's more descriptive.